loop-server/manifest.json

```diff
@@ -2,16 +2,30 @@
   "envs" : {
     "stage": {
       "test_types": [ "stack-check", "loadtest" ],
+      "stack_label": "loopsvrstg",
+      "aws_region": "eu-west-1",
       "urls": {
 	"root" : "loop.stage.mozaws.net",
-	"push_config" : "loop.stage.mozaws.net/push-server-config"
+	"push_config" : "loop.stage.mozaws.net/push-server-config",
+	"heartbeat" : "loop.stage.mozaws.net/__heartbeat__"
+      }
+    },
+    "pre-prod": {
+      "test_types": [ "stack-check" ],
+      "aws_region": "eu-west-1",
+      "urls": {
+        "root": "loop.services.mozilla.com",
+	"push_config" : "loop.services.mozilla.com/push-server-config",
+	"heartbeat" : "loop.services.mozilla.com/__heartbeat__"
```

Comment:
"pre-prod: is essentially prod right before ops switches dns.  i.e. they've deployed an elb to production, but it hasn't gone live yet giving us one final chance to do some testing before it's finally in production.  and of course we wouldn't do any loadtesting in either ;)